cmd/kubeapps-apis/proto/kubeappsapis/core/packages/v1alpha1/packages.proto

```diff
@@ -380,3 +375,38 @@ message Maintainer {
   // A maintainer email
   string email = 2;
   }
+
+  // FilterOptions
+  //
+  // FilterOptions available when requesting summaries
+  message FilterOptions {
+  // TODO: add example for API docs
+  // option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_schema) = {
+  //   example: '{}'
+  // };
+
+    // Text query
+    //
+    // Text query for the request
+    string query = 1;
+
+    // Categories
+    //
+    // Collection of categories for the request
+    repeated string categories = 2;
+
+    // Repositories
+    //
+    // Collection of repositories where the packages belong to
+    repeated string repositories = 3;
+
+    // Package version
+    //
+    // Package version for the request
+    string version = 4;
```

Comment:
I think elsewhere we've stared using `pkg_version` to be specific together with `app_version` as you've used below?

Reply:
Good idea indeed. I've added some comments in the doc for unifying the names here. Using just `version` could be misleading in some cases. I'd change each version by either `pkg_` or `app_` prefixes.